### PR TITLE
Disable play functions in docs mode, unless you set `parameters.docs.autoplay`

### PR DESCRIPTION
**Pull request by tmeasday**
Issue: Play functions cause havoc on docs pages ;)

## What I did

- Don't play unless `runPlayFunction` is set
- Pass that true by default
- Pass that false from the `<Story>` doc block, unless `parameters.docs.autoplay` is set

## How to test

See `autoplay` stories, or e2e/unit tests

### CODE REVIEW

File: code/lib/preview-web/src/render/StoryRender.ts [hunk 1]
```diff
@@ -240,6 +250,7 @@ export class StoryRender<TFramework extends AnyFramework> implements Render<TFra
         this.channel.emit(STORY_RENDERED, id)
       );
     } catch (err) {
+      console.log(err);
```
Comment (shilman) [inline code/lib/preview-web/src/render/StoryRender.ts:253]:
```suggestion
```

**Review by shilman — CHANGES_REQUESTED** [review]
Nice one!!! 🚀

**Review by JReinhold — COMMENTED** [review]
In general I think this is good, but I don't understand why `autoplay` and `runPlayFunction` are different. As far as I understand they do the exact same thing, they're just applied in different places, so I would think it was simpler if they were both called `autoplay`.

**Comment by tmeasday** [comment]
I guess that's a fair point. I guess my thought process was that a pithy name makes more sense for a user facing parameter (`autoplay`) but a longer, more descriptive name made more sense for an internal API.

However, I agree, the confusion of having different names for the same thing is not worth the dubious benefit of the above.

**Comment by tmeasday** [comment]
So the E2E test is failing for `svelte` SBs for a reason I think unrelated to this PR: the "primary" story in docs page isn't rendering properly:
image

Note, oddly, that the rest of the stories do render fine in docspage.

**Review by shilman — APPROVED** [review]
I agree with @JReinhold 's comment on renaming `runPlayFunction` to `autoplay`. Otherwise LGTM!

**Comment by tmeasday** [comment]
Changes are made, this PR is just [blocked on this now](https://github.com/storybookjs/storybook/pull/19659#issuecomment-1296426368).

**Review by JReinhold — APPROVED** [review]
Looks good to me. I'll investigate the Svelte issue. It's been there for a while, I guess it's just now that we're catching it in our tests.

**Comment by tmeasday** [comment]
Thanks for the assist @JReinhold !!

**Pull request merged**